src/shell.zig

```diff
@@ -317,39 +384,89 @@ pub fn git_tag(shell: *Shell) ![]const u8 {
     return stdout;
 }
 
-const Argv = std.ArrayList([]const u8);
+const Argv = struct {
```

Comment:
Argv is changed to suport syntax like `git tag v{version}`, which requires runtime-concatenation of `v` to stringified interpolation variable.